Sources/StreamChat/Config/ChatClientConfig.swift

```diff
@@ -149,8 +149,8 @@ public struct ChatClientConfig {
 
     /// Returns max number of attachments that can be attached to a message.
     ///
-    /// The current limit is `10`.
-    public let maxAttachmentCountPerMessage = 10
+    /// The current limit on the backend is `30`.
```

Comment:
Optional: To be even extra safer, we can make an assertion when creating the `ChatClient`, something like this:
```swift
assert(config.maxAttachmentCountPerMessage < 30)
```

Better yet, we could extract 30 to be a static constant:
```swift
assert(config.maxAttachmentCountPerMessage < ChatClientConfig.maxAttachementCountServerLimit)
```